test/crossbrowser.conf.js

```diff
@@ -154,8 +165,16 @@ exports.config = Object.assign(base, {
           platformVersion: '11.0',
           platformName: 'iOS',
           deviceOrientation: 'portrait',
+
+          max_duration: 180,
+          navigateTimeoutMS: 60000,
+          renderTimeoutMS: 60000,
         },
-    ].filter(
+    ].map(function(c) {
+        // Apply a maximum duration of 1 minute to each test case.
+        c.max_duration = c.max_duration || timeoutSeconds;
```

Comment:
Is this name determined by cbt or by us? It's the only snake_case var in the file

Reply:
It's defined by crossbrowsertesting.com. Wasn't happy about it either. It doesn't even match the selenium/webdriver capabilities format :unamused: